modules/core/MarmotMechanicsCore/include/Marmot/MarmotMaterialHypoElastic.h

```diff
@@ -51,10 +53,36 @@
  *
  * This formulation is compatible with an Abaqus interface.
  */
-class MarmotMaterialHypoElastic : public MarmotMaterialMechanical {
+class MarmotMaterialHypoElastic : public MarmotMaterial {
 
 public:
-  using MarmotMaterialMechanical::MarmotMaterialMechanical;
+  using MarmotMaterial::MarmotMaterial;
+
+  /// Structure to hold the material state at a material point in 3D
+  struct state3D {
+    Marmot::Vector6d stress;       ///> Cauchy stress tensor in Voigt notation
+    double           strainEnergy; ///> Strain energy density
+    double*          stateVars;    ///> Pointer to array of state variables
+  };
+
+  // Structure to hold the material state at a material point for 2D plane stress
+  struct state2D {
+    Marmot::Vector3d stress;       ///> 2D Cauchy stress tensor in Voigt notation
+    double           strainEnergy; ///> Strain energy density
+    double*          stateVars;    ///> Pointer to array of state variables
+  };
+
+  // Structure to hold the material state at a material point for 1D uniaxial stress
+  struct state1D {
+    double  stress;       ///> 1D Cauchy stress
+    double  strainEnergy; ///> Strain energy density
+    double* stateVars;    ///> Pointer to array of state variables
+  };
+
+  struct timeInfo {
```

Comment:
Since the struct only contains 2 doubles: Simply for two arguments instead of the struct?